WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -644,8 +644,8 @@ extension WordPressAppDelegate {
     }
 
     private func checkForAppUpdates() {
-        guard let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String else {
-            DDLogError("No CFBundleShortVersionString found in Info.plist")
+        guard let version = Bundle.main.infoDictionary?["CFBundleVersion"] as? String else {
+            DDLogError("No CFBundleVersion found in Info.plist")
```

Comment:
(Nit) A good place for `wpAssertionFailure`.

Reply:
Fixed! [abb2724](https://github.com/wordpress-mobile/WordPress-iOS/pull/23224/commits/abb27246f3ebca40adeaecf46e3b39fbbf5c250a)